### PR TITLE
Add numberWithCommas

### PR DESCRIPTION
**Pull request by feiwen8772**
Use commas to separate digits。

### CODE REVIEW

**Comment by feiwen8772** [comment]
@iamsoorena   Can you help me check the English description?

**Comment by iamsoorena** [comment]
@feiwen8772 sure I would love to do that. just give me few minutes to reach my laptop.

File: snippets/numberWithCommas.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### numberWithCommas
+
+Use commas to separate digits。
```
Comment (Pl4gue) [inline snippets/numberWithCommas.md]: Convert a float-point arithmetic to the [Decimal mark](https://en.wikipedia.org/wiki/Decimal_mark) form.

Comment (Pl4gue) [inline snippets/numberWithCommas.md]: Change this line to: 

Convert a float-point arithmetic to the [Decimal mark](https://en.wikipedia.org/wiki/Decimal_mark) form.

File: snippets/numberWithCommas.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Use `toString()` to convert the number `num` to a string, then use `/\B(?=(\d{3})+(?!\d))/g` to separate every three characters with a comma,except word boundary。
```
Comment (Pl4gue) [inline snippets/numberWithCommas.md]: And change this sentence to:

Use `toString()` to convert the number `num` to a string, then use regex to separate every three characters of the integer part with a comma.

**Comment by iamsoorena** [comment]
@Pl4gue @feiwen8772  Oh that was so fast. I just got behind my desk. Happy that problems are resolved now.

**Comment by feiwen8772** [comment]
@Pl4gue @iamsoorena 👍

**Comment by iamsoorena** [comment]
thank you @feiwen8772 for your contribution.

**Pull request merged**